specification/azsadmin/resource-manager/compute/Microsoft.Compute.Admin/preview/2018-07-30-preview/Disks.json

```diff
@@ -44,6 +44,12 @@
           {
             "$ref": "Disks.json#/parameters/SourceSharePathParameter"
           },
+          {
```

Comment:
@zyxyoshine Havent you made this change already to master? you made this only in stackadmin?

Reply:
@bganapa I believe they are in master. PR https://github.com/Azure/azure-rest-api-specs/pull/8747 has been merged.